cmd/nginx/flags.go

```diff
@@ -149,15 +146,22 @@ Feature backed by OpenResty Lua libraries. Requires that OCSP stapling is not en
 
 		enableMetrics = flags.Bool("enable-metrics", true,
 			`Enables the collection of NGINX metrics`)
+		metricsPerHost = flags.Bool("metrics-per-host", true,
```

Comment:
It was always enabled before, since default value is true this doesn't change anything for us